hbshelpers/sdkAssetUrl.js

```diff
@@ -0,0 +1,35 @@
+const RELEASE_BRANCH_REGEX = /^release\/v[0-9.]+$/;
+const HOTFIX_BRANCH_REGEX = /^hotfix\/v[0-9.]+$/;
+const SEM_VER_REGEX = /^[1-9]+$|^[1-9]+\.[0-9]+$|^[1-9]+\.[0-9]+\.[0-9]+$/;
```

Comment:
are we intentionally leaving out SDK versions before 1.0 because nobody should be using them anyway?

Reply:
We don't support pre-1.0 versions any more.  don't think they would work with the Theme.